firefox-ios/firefox-ios-tests/Tests/XCUITests/BaseTestCase.swift

```diff
@@ -512,6 +511,13 @@ extension XCUIElement {
         let visibleScreenFrame = getVisibleScreenFrame(app: app)
         return self.exists && isPartiallyIncluded(rectangleArea: visibleScreenFrame, rectangleToBeIncluded: self.frame)
     }
+    /// Waits for the UI element and then taps it if it exists.
+    func waitAndTap(timeout: TimeInterval? = TIMEOUT) {
+        BaseTestCase().mozWaitForElementToExist(self, timeout: timeout)
+        if self.exists {
```

Comment:
same comment I did before

Reply:
Updated `mozWaitForElementToExist` function to use `guard`. Thanks for the tip 🙏🏽 
Fixed here: https://github.com/mozilla-mobile/firefox-ios/pull/22528/commits/a50c8636d5aaee960a6340cd482fff64644695fc